akka24-conductr-client-lib/src/main/scala/com/typesafe/conductr/clientlib/akka/BundlesConnector.scala

```diff
@@ -0,0 +1,298 @@
+   *                                   from `/bundles/events`
+   * @param reconnectInterval the amount of time to wait to reconnect to `/bundles/events` when disconnected.
+   * @param system the actor system which is used to build [[BundlesConnector]]
+   * @return the flow graph of `Flow[(HttpRequest, HttpRequest), Seq[Bundle], NotUsed]`.
+   */
+  def connect(
+    conductrAddress: URL,
+    stopAfter: FiniteDuration,
+    expectingEventBurstTimeout: FiniteDuration = 500.millis,
+    reconnectInterval: FiniteDuration = 500.millis
+  )(implicit system: ActorRefFactory): Flow[(HttpRequest, HttpRequest), Seq[Bundle], NotUsed] =
+    Flow[(HttpRequest, HttpRequest)].flatMapConcat {
+      case (getBundlesEventsRequest, getBundlesRequest) =>
+        Source.actorPublisher(props(conductrAddress, getBundlesEventsRequest, getBundlesRequest, Some(stopAfter), expectingEventBurstTimeout, reconnectInterval))
+    }
+
+  /**
+   * Client API which provides a nicer interface to the [[BundlesConnector]] actor.
+   *
+   * Provides a means for the caller to connect to the ConductR Bundles endpoint continuously.
+   *
+   * Returns a flow graph of `Flow[(HttpRequest, HttpRequest), Seq[Bundle], NotUsed]`.
+   *
+   * The input is a tuple of [[HttpRequest]]:
+   * - The first element of the tuple is the HTTP request for ConductR Bundles Events endpoint.
+   * - The second element of the tuple is the HTTP request for ConductR Bundles endpoint.
+   *
+   * The `Seq[Bundle]` will be emitted whenever there's any change of the bundle state within ConductR.
+   * The change detection is derived from having Bundle SSE propagated by ConductR Bundles Events endpoint, and an
+   * updated `Seq[Bundle]` being recognized by the [[BundlesConnector]] actor.
+   *
+   * @param conductrAddress the ConductR Control Protocol base URL
+   * @param expectingEventBurstTimeout the expected amount of time of which SSE event burst from `/bundles/events` will
+   *                                   take place. Once event burst is over, the latest bundle state will be obtained
+   *                                   from `/bundles/events`
+   * @param reconnectInterval the amount of time to wait to reconnect to `/bundles/events` when disconnected.
+   * @param system the actor system which is used to build [[BundlesConnector]]
+   * @return the flow graph of `Flow[(HttpRequest, HttpRequest), Seq[Bundle], NotUsed]`.
+   */
+  def continuously(
+    conductrAddress: URL,
+    expectingEventBurstTimeout: FiniteDuration = 500.millis,
+    reconnectInterval: FiniteDuration = 500.millis
+  )(onBundleUpdates: Seq[Bundle] => Unit)(implicit system: ActorRefFactory): Flow[(HttpRequest, HttpRequest), Seq[Bundle], NotUsed] =
+    Flow[(HttpRequest, HttpRequest)].flatMapConcat {
+      case (getBundlesEventsRequest, getBundlesRequest) =>
+        Source.actorPublisher(props(conductrAddress, getBundlesEventsRequest, getBundlesRequest, None, expectingEventBurstTimeout, reconnectInterval))
+    }
```

Comment:
This function feels as though it can be combined with the other now i.e. we don't need two. The only diff is the one param. Perhaps expose the param as a function param.